CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # SAS Viya Monitoring for Kubernetes
 
+## Unreleased
+* **Overall**
+  * [FEATURE] Deploying into an air-gapped environment using local Helm archive (.tgz) files is now supported.  In this scenario,
+container images are still expected to be pulled from private container registry.  See
+[Configure SAS Viya Monitoring for Kubernetes for an Air-Gapped Environment](https://documentation.sas.com/?cdcId=obsrvcdc&cdcVersion=default&docsetId=obsrvdply&docsetTarget=n0grd8g2pkfglin12bzm3g1oik2p.htm).
+  * [CHANGE] The environment variables over-riding the default Helm chart versions to deploy MUST now be set in the user.env files
```

Comment:
I have very mixed feelings about including this 2nd item in the CHANGELOG.  On the one hand, our move to the component_versions.env file does require/implement this change.  On the other hand, I don't believe we've documented this set of environment variables, so no one is likely using them.  Thoughts?

Reply:
We probably don't need to include the second bullet point - I don't think there's a good reason for users to change it and I wouldn't want someone to change it and cause their regular, online deployments to fail.

Reply:
I have dropped the 2nd bullet point in my latest round of commits.